Graphviz/DOT.tmPreferences

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why messing everything in a single file?

Reply:
All packages use a common scheme of how to group and name preference files and I wouldn't change it! There's no reason to change everything to 180 degree!